content/docs/install/linux.md

```diff
@@ -9,20 +9,17 @@
 > [virtual environment](https://python.readthedocs.io/en/stable/library/venv.html)
 > or using
 > [pipx](https://packaging.python.org/guides/installing-stand-alone-command-line-tools/)
-> (on Python 3.7+) to encapsulate your local environment.
+> (on Python 3.7+) to encapsulate your local environment.  
+> Note that `pip` version 19+ is required.
```

Comment:
Also, is pip 19 still the min. required here? Per https://github.com/iterative/dvc.org/issues/2112

Reply:
I was confused, #2112 related to setting up a dev env. I rolled this back. Or should we specify a `pip` version? I remember regular `pip` installation failing with the default `pip` in Ubuntu before because it was pretty old.